pkg/ctl/install/pki.go

```diff
@@ -0,0 +1,167 @@
+package install
+
+import (
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/cloudflare/cfssl/cli/genkey"
+	"github.com/cloudflare/cfssl/config"
+	"github.com/cloudflare/cfssl/csr"
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/cloudflare/cfssl/initca"
+	"github.com/cloudflare/cfssl/log"
+	"github.com/cloudflare/cfssl/signer"
+	"github.com/cloudflare/cfssl/signer/local"
+)
+
+type publicKeyInfrastructurePaths struct {
+	caCertificate     string
+	caKey             string
+	serverCertificate string
+	serverKey         string
+	clientCertificate string
+	clientKey         string
+}
+
+type publicKeyInfrastructure struct {
+	caCertificate     []byte
+	caKey             []byte
+	serverCertificate []byte
+	serverKey         []byte
+	clientCertificate []byte
+	clientKey         []byte
+}
+
+type devnull struct{}
+
+func (devnull) Debug(string)   {}
+func (devnull) Info(string)    {}
+func (devnull) Warning(string) {}
+func (devnull) Err(string)     {}
+func (devnull) Crit(string)    {}
+func (devnull) Emerg(string)   {}
+func init() {
+	// Disable logging of cfssl since we handle errors explicitly
+	log.SetLogger(devnull{})
```

Comment:
Does it have to be an init function?

Reply:
Where would you put it?

Reply:
We only use it one place, just having it in the entry point function would be fine.

Reply:
My answer got lost, sorry. I mentioned that there isn't an entry point function as such.